tron/kubernetes.py

```diff
@@ -281,10 +282,12 @@ def __init__(
         default_volumes: Optional[List[ConfigVolume]] = None,
         pod_launch_timeout: Optional[int] = None,
         watcher_kubeconfig_paths: Optional[List[str]] = None,
+        non_retryable_exit_codes: Optional[Tuple[int, ...]] = (),
```

Comment:
I dont think we really _want_ this to be a Tuple; the validator validates that we have a _list_ of this type instead. 

If your mypy was telling you you needed this to be a Tuple instead, it is likely simply because you are setting defaults everywhere to tuples rather than the empty list :D